app/components/ContextMenu.tsx

```diff
@@ -176,6 +177,37 @@ const OptionViewArtist = React.memo<{
   )
 })
 
+const OptionShuffleArtist = React.memo<{
+  artistId?: string
```

Comment:
This can be non-optional though right? The artist ID should always be there if we're popping up an artist context menu already.

Reply:
I was just following the pattern I saw in `OptionViewArtist`, but your point is valid so I've removed this.